extensions/frontend/pkg/command/command.go

```diff
@@ -53,13 +53,6 @@ func Frontend(cfg *config.Config) *cli.Command {
 			uuid := uuid.Must(uuid.NewV4())
 			pidFile := path.Join(os.TempDir(), "revad-"+c.Command.Name+"-"+uuid.String()+".pid")
 
-			// pregenerate list of valid localhost ports for the desktop redirect_uri
-			// TODO use custom scheme like "owncloud://localhost/user/callback" tracked in
-			var desktopRedirectURIs [65535 - 1024]string
-			for port := 0; port < len(desktopRedirectURIs); port++ {
-				desktopRedirectURIs[port] = fmt.Sprintf("http://localhost:%d", (port + 1024))
-			}
-
```

Comment:
oh this is no longer necessary? I see that `desktopRedirectURIs` isn't used anywhere ... but IIRC ... ahhhh this was used for the basic reva oidc provider ...